### ISSUE TITLE
Flag passwords as sensitive

### ISSUE THREAD

**Issue by MarshallAsch** [maxlaverse/terraform-provider-bitwarden#14]
The passwords that are loaded from bitwarden using the plugin are not marked as sensitive in the output of the various terraform commands. 

### Terraform Version

- Terraform: `1.1.4`
- bitwarden provider: `0.1.0`

### Affected Resource(s)

I noticed the issue with `data "bitwarden_item_login"`, but I suspect it is present in all resources. 

### Terraform Configuration Files
```hcl
data "bitwarden_item_login" "mysql-credentials" {
  id = var.db_credentials_id
  sensitive = true
}

resource "aws_db_instance" "application_db" {
  username                        = data.bitwarden_item_login.mysql-credentials.username
  password                        = data.bitwarden_item_login.mysql-credentials.password
  ....
}

```

### Expected Behavior
I would expect that the `username` and `password` fields, or at least just the `password` field to be marked as sensitive in the output of the `plan` command. 

### Actual Behavior

Output from `terraform plan`
```
  # aws_db_instance.application_db will be updated in-place
  ~ resource "aws_db_instance" "application_db" {
      # Warning: this attribute value will no longer be marked as sensitive
      # after applying this change. The value is unchanged.
      ~ password                              = (sensitive value)
      # Warning: this attribute value will no longer be marked as sensitive
      # after applying this change. The value is unchanged.
      ~ username                              = (sensitive)
        # (43 unchanged attributes hidden)
...
```

The result is both of these variables are no longer marked as sensitive in the output. 

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform plan`

### Important Factoids
There is nothing special about this terraform configuration, it is being migrated away from plain variables marked as sensitive to load passwords from bitwarden. 

### References
The documentation seems to mention that the output fields could be marked as `sensitive` through the use of the sensitive flag or the sensitive function https://www.terraform.io/plugin/sdkv2/best-practices/sensitive-state

**Comment by maxlaverse** [comment]
Hi @MarshallAsch,
Indeed, this is a bug. Some of those fields are marked as sensitive but only in the *resources*, not the *data sources*. I'll try to release a new version as soon as possible with the following field marked as sensitive:
* username
* password
* notes
* totp
* fields (because another user reported to use fields to store API keys, https://github.com/maxlaverse/terraform-provider-bitwarden/issues/8)

Thanks for the report

**Comment by MarshallAsch** [comment]
Are you okay if I make a PR with the change? I think it should be fairly simple and would be a nice little intro to go for me.

**Comment by maxlaverse** [comment]
Of course! You would probably be the first besides me to run the project locally, so don’t hesitate to ping me if you feel some steps are missing in the README, or if it’s simply not working.

On 21. Mar 2022, at 21:15, Marshall Asch ***@***.***> wrote:

﻿

Are you okay if I make a PR with the change? I think it should be fairly simple and would be a nice little intro to go for me.

**Comment by MarshallAsch** [comment]
@maxlaverse the PR is up for this now, let me know if there are any other improvements / changes that you need.

**Comment by maxlaverse** [comment]
Thanks for your contribution @MarshallAsch !
I released it under version v0.1.1.
